<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources_file.qrc">
    <normaloff>:/Box.png</normaloff>:/Box.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	background-color: #efefef;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>183</y>
      <width>1260</width>
      <height>557</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGraphicsView {
	background-color: rgb(255, 255, 255);
	border: 0px solid ;
    border-radius: 20px;
}</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>118</y>
      <width>125</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
	background-color: #ffffff;
	color: rgb(54, 54, 54);
	font: 500 10pt &quot;JetBrains Mono&quot;;
	border-radius: 5px;
}

QComboBox::drop-down { border-width: 0px; };
QComboBox::down-arrow { image: none; border-width: 0px; };</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>95</y>
      <width>124</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 12pt &quot;JetBrains Mono&quot;;
}</string>
    </property>
    <property name="text">
     <string>Фигура</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>193</x>
      <y>0</y>
      <width>1106</width>
      <height>173</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>92</x>
        <y>15</y>
        <width>83</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 12pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>Создать</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>80</y>
        <width>239</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { 
	background-color: rgb(255, 255, 255);
	border: 5px solid #ebebeb;
    border-radius: 19px; 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>80</y>
        <width>511</width>
        <height>80</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { 
	background-color: rgb(255, 255, 255);
	border: 5px solid #ebebeb;
    border-radius: 19px; 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>74</x>
        <y>41</y>
        <width>118</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { 
	background-color: rgb(255, 255, 255);
	border: 0px solid;
    border-radius: 8px; 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>473</x>
        <y>41</y>
        <width>69</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel { 
	background-color: rgb(255, 255, 255);
	border: 0px solid;
    border-radius: 8px; 
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>89</x>
        <y>46</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 12pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>Колич.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>482</x>
        <y>46</y>
        <width>19</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 12pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>№</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>99</y>
        <width>89</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>Координаты</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>159</x>
        <y>99</y>
        <width>63</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>Размеры</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_count">
      <property name="geometry">
       <rect>
        <x>156</x>
        <y>46</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_index_size">
      <property name="geometry">
       <rect>
        <x>505</x>
        <y>46</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ok_create">
      <property name="geometry">
       <rect>
        <x>54</x>
        <y>148</y>
        <width>67</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	background-color: #FFFFFF;
	border: 1px solid #ebebeb;
    border-radius: 5px;
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources_file.qrc">
        <normaloff>:/Check.png</normaloff>:/Check.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_random_create">
      <property name="geometry">
       <rect>
        <x>145</x>
        <y>148</y>
        <width>67</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	background-color: #FFFFFF;
	border: 1px solid #ebebeb;
    border-radius: 5px;
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources_file.qrc">
        <normaloff>:/Refresh cw.png</normaloff>:/Refresh cw.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_random_size">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>148</y>
        <width>67</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	background-color: #FFFFFF;
	border: 1px solid #ebebeb;
    border-radius: 5px;
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources_file.qrc">
        <normaloff>:/Refresh cw.png</normaloff>:/Refresh cw.png</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_ok_size">
      <property name="geometry">
       <rect>
        <x>469</x>
        <y>148</y>
        <width>67</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	background-color: #FFFFFF;
	border: 1px solid #ebebeb;
    border-radius: 5px;
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources_file.qrc">
        <normaloff>:/Check.png</normaloff>:/Check.png</iconset>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>88</y>
        <width>87</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>текущие</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>444</x>
        <y>88</y>
        <width>44</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>новые</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>591</x>
        <y>88</y>
        <width>62</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>текущие</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>695</x>
        <y>88</y>
        <width>86</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>смещение</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_x">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>120</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_y">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>120</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px;
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff; 
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_w">
      <property name="geometry">
       <rect>
        <x>148</x>
        <y>120</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_h">
      <property name="geometry">
       <rect>
        <x>198</x>
        <y>120</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_current_w">
      <property name="geometry">
       <rect>
        <x>303</x>
        <y>109</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_current_h">
      <property name="geometry">
       <rect>
        <x>353</x>
        <y>109</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_new_w">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>109</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_new_h">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>109</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_current_x">
      <property name="geometry">
       <rect>
        <x>576</x>
        <y>109</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_current_y">
      <property name="geometry">
       <rect>
        <x>626</x>
        <y>109</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_dx">
      <property name="geometry">
       <rect>
        <x>692</x>
        <y>109</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_dy">
      <property name="geometry">
       <rect>
        <x>742</x>
        <y>109</y>
        <width>42</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit { 
	background-color: #d4d4d4;
	border: 0px solid;
    border-radius: 5px; 
	font: 9pt &quot;JetBrains Mono&quot;;
	color: #ffffff;
}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search_size">
      <property name="geometry">
       <rect>
        <x>549</x>
        <y>41</y>
        <width>38</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton { 
	background-color: rgb(255, 255, 255);
	border: 0px solid;
    border-radius: 8px; 
}

</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources/resources_file.qrc">
        <normaloff>:/Search.png</normaloff>:/Search.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>23</width>
        <height>23</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>308</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>w</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>h</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>478</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>h</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>426</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>w</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>582</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>x</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>634</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>y</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>697</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>dx</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>749</x>
        <y>130</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 11pt &quot;JetBrains Mono&quot;;
}</string>
      </property>
      <property name="text">
       <string>dy</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_all_size">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>43</y>
        <width>55</width>
        <height>27</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	color: rgb(54, 54, 54);
	font: 500 12pt JetBrains Mono;
}</string>
      </property>
      <property name="text">
       <string>ВСЕ</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>1191</x>
      <y>165</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 9pt &quot;JetBrains Mono&quot;;
}</string>
    </property>
    <property name="text">
     <string>W: 1260</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>739</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: rgb(54, 54, 54);
	font: 500 9pt &quot;JetBrains Mono&quot;;
}</string>
    </property>
    <property name="text">
     <string>H: 557</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/resources_file.qrc"/>
 </resources>
 <connections/>
</ui>
